specifications_dbus.md

```diff
@@ -0,0 +1,92 @@
+
+## Connector API
+
+Connectors MUST provide a service with their application ID, that is, the same name as the name that they sent their `org.unifiedpush.Distributor1.Register` call from.
+
+Connectors MUST implement the `org.unifiedpush.Connector1` interface at the object path `/org/unifiedpush/Connector`. The caller of the methods in this interface MUST NOT wait for a response from them!
+
+### org.unifiedpush.Connector1.Message (String, String, String) → nothing
+
+The distributor calls this method to send a new push message to the connector.
+
+The first argument is the token of the connection, the second argument is the raw POST data as received by the distributor, and the third argument is optionally an ID identifying the message (otherwise it is an empty string). This method does not return anything.
+
+### org.unifiedpush.Connector1.NewEndpoint (String, String) → nothing
+
+The distributor calls this method to inform the connector of the endpoint URL, both after registering for the first time, and if the endpoint changes afterwards. In addition to implementing this method, the connector SHOULD also call [org.unifiedpush.Distributor1.Register](#orgunifiedpushdistributor1register-string--string-string) on every startup to request the newest endpoint URL, in case it missed this call otherwise.
+
+The first argument is the token of the connection, and the second argument is the new endpoint. This method does not return anything.
+
+### org.unifiedpush.Connector1.Unregister (String) → nothing
+
+The distributor calls this method to inform the connector that it has been unregistered for some server-side or distributor-side reason. If the app wants to remain registered, it SHOULD simply call [org.unifiedpush.Distributor1.Register](#orgunifiedpushdistributor1register-string--string-string) again after getting unregistered. A response indicating "REGISTRATION_REFUSED" would mean that the app is no longer allowed to register.
+
+The argument of the method is the token of the connection. This method does not return anything.
+
+## Distributor
+
+A *distributor* is a service which runs on the device, receiving push messages via whatever transport it wants, and sends them on to the connector using the connector API.
+
+## Distributor API
+
+Distributors MUST provide a service with a name beginning with `org.unifiedpush.Distributor.` followed by the name of the distributor (for example, `org.unifiedpush.Distributor.distributorname` where `distributorname` is the distributor's name)
+
+The distributor MUST implement the `org.unifiedpush.Distributor1` interface at the object path `/org/unifiedpush/Distributor`.
+
+### org.unifiedpush.Distributor1.Register (String, String) → (String, String)
+
+The connector calls this method to register for push messages, or to retreive the newest push endpoint (as it may change at any time). See also the `org.unifiedpush.Connector1.NewEndpoint` method.
```

Comment:
It should follow a bit this scheme
https://github.com/UnifiedPush/specifications/blob/main/specifications/android.md#orgunifiedpushandroiddistributorregister